src/init.c

```diff
@@ -403,6 +403,30 @@ int hyper_enter_sandbox(struct hyper_pod *pod, int pid_efd)
 	return ret;
 }
 
+/*
+ * All containers in the pod share the same ipc namespace. However,
+ * posix ipc primitives are shm_open() family whose behaviors
+ * implemented in glibc are to create&share the shm objects within
+ * /dev/shm (or scans /proceed/mounts for any tmpfs if /dev/shm
+ * is not tmpfs).
+ * So we have to create the only one tmpfs mount and share it
+ * to all the containers.
+ */
+static int hyper_setup_shm(struct hyper_pod *pod)
+{
+	if (hyper_mkdir("/tmp/hyper/shm", 0755) < 0) {
```

Comment:
Is it good to be placed in `/tmp`? how about `/var/lib` or somewhere like?

Reply:
in hyperstart, all pod/containers related things are created on /tmp/hyper